websites/views/blogs_summary.view.lkml

```diff
@@ -0,0 +1,10 @@
+include: "//looker-hub/websites/views/blogs_daily_summary.view.lkml"
+
+view: blogs_summary {
+  extends: [blogs_daily_summary]
+
+  filter: date {
```

Comment:
I don't think I've seen this in other explores. What does it do?

Reply:
Since I'm not very familiar with Looker I just grabbed that from another existing view:
https://github.com/mozilla/looker-spoke-default/blob/main/user_journey/views/cohort_analysis/cohort_analysis.view.lkml#L6-L9

In Looker this is what I see:
![image](https://user-images.githubusercontent.com/42538694/168264542-523d59be-43b7-49c4-ad21-ca4d2521e9ff.png)

Reply:
![image](https://user-images.githubusercontent.com/42538694/168264937-f4458540-ac4d-47cd-b168-9e512de3060a.png)


So this seems as a way for defining a filter field. Original example was casting to `timestamp`, looks like that might be needed:
![image](https://user-images.githubusercontent.com/42538694/168265123-03995a91-53c8-4a8b-9d51-d0f014f20fc4.png)

Reply:
This is what it looks like when you click on the filter field in Looker UI: 
![image](https://user-images.githubusercontent.com/42538694/168265234-e1c5bf01-fdf5-421a-bb2f-a05a4817cbd9.png)